### PR TITLE
Printable id card

### PR DESCRIPTION
**Pull request by Mudassar045**
[Click Here to See - Demo](https://www.dropbox.com/s/9sf7t1odizrf5k6/demo-printable-id-cards.pdf?dl=0)

### CODE REVIEW

File: web/src/components/Printable/Student/card.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+const StudentIDCard = (props: PropsTypes) => {
+    return(<div className="student-card">  
+        <div className="card-row card-school-info">
+            <img className="card-school-logo" src={props.schoolLogo} alt="School Logo"/>
+            <div className="card-school-title">{props.schoolName}</div>
+        </div>
+        <div className="card-row first" >
+            <div>
+                <img className="card-student-profile" src="https://via.placeholder.com/100" alt="student-profile"/>
```
Comment (taimur38) [inline web/src/components/Printable/Student/card.tsx]: good placeholder for now but should replace this with background image and a fixed height/width. that way we don't make requests to this site in production

File: web/src/components/Printable/Student/card.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            </div>
+            <div className="card-student-info">
+                <div>Name: <span>{ props.student.Name }</span></div>
+                <div>class: <span>{ props.studentClass }</span></div>
+                <div>Roll No:<span>{ props.student.RollNumber }</span></div>
```
Comment (taimur38) [inline web/src/components/Printable/Student/card.tsx]: should have the space here also. and "class" should be capitalized

File: web/src/components/Printable/Student/card.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            </div>
+        </div>
+        <div className="card-row last">
+            <div className="card-column">
+                <div className="card-signature">Issuing Authority</div>
+            </div>
+            <div className="card-column">
+                <div className="card-valid-date">Valid { moment().format("YYYY") }-{ moment().add(1, "year").format("YYYY") }</div>
```
Comment (taimur38) [inline web/src/components/Printable/Student/card.tsx]: School owners may challenge this but ok for now

File: web/src/components/Printable/Student/cardlist.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import "./style.css"
+
+type PropsTypes = {
+  students: MISStudent[]
+  schoolName: string
+  schoolLogo: string
+  studentClass: string
+}
+
+export const StudenPrintabletIDCardList = (props: PropsTypes) => {
```
Comment (taimur38) [inline web/src/components/Printable/Student/cardlist.tsx]: typo - should be `StudentPrintableIDCardList`

File: web/src/components/Printable/Student/card.tsx (hunk 1 above)
Comment (Mudassar045) [reply web/src/components/Printable/Student/card.tsx]: height/width is fixed 100x100. Yes will replace it with student profile

File: web/src/components/Printable/Student/card.tsx (hunk 2 above)
Comment (Mudassar045) [reply web/src/components/Printable/Student/card.tsx]: will change it

**Pull request merged**